src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -93,6 +101,66 @@ export class ZerionBalancesApi implements IBalancesApi {
     }
   }
 
+  /**
+   * NOTE: Zerion does not support limit & offset parameters.
+   *
+   * It uses a "size" query param for the page size, and an "after" parameter for the offset.
+   * "size" is an integer which could be mapped to "limit", but "after" is a base64-encoded string.
```

Comment:
Are there docs about this that we could add a link to?

Reply:
Absolutely! Thanks!

[1a730ce](https://github.com/safe-global/safe-client-gateway/pull/1121/commits/1a730ce238b8101cd84728a16c784176c139b255)